pushmsixscript/tests/test_artifacts.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+from scriptworker import artifacts
```

Comment:
You know the drill :)

As an aside on tests: 93% coverage, nice! I prefer 100% but we're far from getting there on the scripts repo. Close on the scriptworker repo.

I did see this:

```
ERROR: invocation failed (exit code 1), logfile: /builds/worker/checkouts/src/pushmsixscript/.tox/coveralls/log/coveralls-2.log
[task 2021-11-29T20:39:22.203Z] ================================== log start ===================================
[task 2021-11-29T20:39:22.203Z] Obtaining file:///builds/worker/checkouts/src/pushmsixscript
[task 2021-11-29T20:39:22.203Z]   Installing build dependencies: started
[task 2021-11-29T20:39:22.203Z]   Installing build dependencies: finished with status 'done'
[task 2021-11-29T20:39:22.203Z]   Checking if build backend supports build_editable: started
[task 2021-11-29T20:39:22.203Z]   Checking if build backend supports build_editable: finished with status 'done'
[task 2021-11-29T20:39:22.204Z]   Getting requirements to build wheel: started
[task 2021-11-29T20:39:22.204Z]   Getting requirements to build wheel: finished with status 'done'
[task 2021-11-29T20:39:22.204Z]   Preparing metadata (pyproject.toml): started
[task 2021-11-29T20:39:22.204Z]   Preparing metadata (pyproject.toml): finished with status 'done'
[task 2021-11-29T20:39:22.204Z] Collecting taskcluster
[task 2021-11-29T20:39:22.204Z]   Using cached taskcluster-44.2.2-py3-none-any.whl (135 kB)
[task 2021-11-29T20:39:22.204Z] ERROR: Could not find a version that satisfies the requirement scriptworker-client (from pushmsixscript) (from versions: none)
[task 2021-11-29T20:39:22.204Z] ERROR: No matching distribution found for scriptworker-client
[task 2021-11-29T20:39:22.204Z] 
[task 2021-11-29T20:39:22.204Z] =================================== log end ====================================
[task 2021-11-29T20:39:22.204Z] ___________________________________ summary ____________________________________
[task 2021-11-29T20:39:22.204Z] ERROR:   coveralls: InvocationError for command /builds/worker/checkouts/src/pushmsixscript/.tox/coveralls/bin/python -m pip install --exists-action w -e . (exited with code 1)
[task 2021-11-29T20:39:22.223Z]
```